hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java

```diff
@@ -467,7 +541,7 @@ boolean processKeyPath(List<String> keyPathList) {
       for (String keyPath : keyPathList) {
         OFSPath path = new OFSPath(keyPath);
         OzoneManagerProtocolProtos.OMRequest omRequest =
-            getDeleteKeyRequest(path);
+            getDeleteKeysRequest(path);
```

Comment:
Clarification: Is this related to the FSO bucket? This is required for the un optimized bucket, right?

Reply:
Yes `DeleteKeysRequest `is for un optimised bucket. Using `DeleteKeyRequest` for unoptimised bucket would cause [this](https://github.com/apache/ozone/pull/1851) issue.

Reply:
OK, got it. 

I have gone through the master code and saw it used [getDeleteKeyRequest#470](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java#L470)

Should we fix this in master ?

Reply:
Currently the master code has one method for rename and one method for delete.  I have added getDeleteKeyRequest and getDeleteKeysRequest in this branch inorder to differentiate between the 2 delete requests.